perseo-utils/src/main/java/com/telefonica/iot/perseo/utils/DateTimeUtils.java

```diff
@@ -93,6 +94,44 @@ public static long getMilisToNextSunrise(Calendar day, double latitude, double l
         return (sunriseSunrise.getTimeInMillis() - day.getTimeInMillis());
     }
 
+    /**
```

Comment:
I understand the changes in this PR are covered in CHANGES_NEXT_RELEASE by the following entry:

```
Add perseo-utils library import into Esper
```

I wonder if it would be a good idea edit that entry to provide some information about the kind of utils added. For instance:

```
Add perseo-utils library (date time utility functions) into Esper
```

or something similar.

Reply:
As you suggested, that line was edited on CNR file.

Fixed in [7a4b6fc](https://github.com/telefonicaid/perseo-core/pull/150/commits/7a4b6fc279e8676e72e7bcdc125552a7f3d84609)